pkg/controller/account/account_controller.go

```diff
@@ -407,11 +406,11 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 			}
 		}
 
-		// see if in creating for longer then 10 minutes
+		// see if in creating for longer then 25 minutes
 		now := time.Now()
 		diff := now.Sub(currentAcctInstance.ObjectMeta.CreationTimestamp.Time)
-		if currentAcctInstance.Status.State == "Creating" && diff > createPendTime {
-			r.setStatusFailed(reqLogger, currentAcctInstance, "Creation pending for longer then 10 minutes")
+		if currentAcctInstance.Status.State == "Creating" && diff > controllerutils.WaitTime * time.Minute {
```

Comment:
Can we pull this "calculation" out?  (I don't specifically know what to call it).  Calculations inside complex if statements make the code harder to grok the second time around.

Maybe we should pull this `controllerutils.WaitTime * time.Minute` part out and just put it on the line above, and maybe call it something like `createTimeout`?

Reply:
Agree. Put the expression (waitTime * time.Minute) as the definition of createPendTime and leave this line as diff > createPendTime

Reply:
Done